src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,48 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.TaskList;
+import duke.Ui;
+import duke.task.Deadline;
+import duke.task.Task;
+
+public class DeadlineCommand extends Command {
+    private String taskName;
+    private String deadlineString;
```

Comment:
deadlineString might be misleading as it could mean the description of the task. A more appropriate name would be deadlineDateAndTime.